assignments/4. operators/loops.md

```diff
@@ -32,20 +64,44 @@ var amount = 0;
 "1 * 4 = 4"
 .... for all 100 results)
 
+```js
+    for (let r = 1; r <= 10; r++ ) {
+        for (let p = 1; p <= 10; p++) {
+            console.log(`${r} * ${p} = ${r*p}`);
+        }
+    }
+```
+
 5. 🎖Show the following output using one loop.
 ```js
 // 1, 2, 3, 4, 5
 // 6, 7, 8, 9, 10
 
 // Your code goes here
+for (let k = 1; k <= 10; k++) {
+    console.log(k);
+}
```

Comment:
The output is one after another.

<img width="333" alt="Screenshot 2019-06-22 at 12 15 49" src="https://user-images.githubusercontent.com/3122626/59960564-96cec400-94e7-11e9-98d6-c86177ba8ed8.png">

It should be same as it was shown there. One next to other